third_party/txt/src/txt/font_collection.cc

```diff
@@ -250,6 +250,28 @@ void FontCollection::SortSkTypefaces(
         int a_delta = std::abs(a_style.width() - SkFontStyle::kNormal_Width);
         int b_delta = std::abs(b_style.width() - SkFontStyle::kNormal_Width);
 
+        {
+          // A workaround to prevent emoji fonts being selected for normal text
+          // when normal and emojis are mixed at same font family.
```

Comment:
You may move this code block before line 247.

Reply:
Is line 247 the original part?

Reply:
Before the declaration of `a_style`, because none of `a_style`, ..., `b_delta` is used in this block.

Reply:
Oh.. It turns out I didn't understand your intent. I thought you were talking about the comment section only!